README.md

```diff
@@ -95,18 +142,18 @@ If your project does not yet have a Podfile, use the `pod init` command in the r
 use_frameworks!
 
 target 'MyApp' do
-    pod 'IBMWatsonAssistantV1', '~> 4.0.1'
-    pod 'IBMWatsonAssistantV2', '~> 4.0.1'
-    pod 'IBMWatsonCompareComplyV1', '~> 4.0.1'
-    pod 'IBMWatsonDiscoveryV1', '~> 4.0.1'
-    pod 'IBMWatsonLanguageTranslatorV3', '~> 4.0.1'
-    pod 'IBMWatsonNaturalLanguageClassifierV1', '~> 4.0.1'
-    pod 'IBMWatsonNaturalLanguageUnderstandingV1', '~> 4.0.1'
-    pod 'IBMWatsonPersonalityInsightsV3', '~> 4.0.1'
-    pod 'IBMWatsonSpeechToTextV1', '~> 4.0.1'
-    pod 'IBMWatsonTextToSpeechV1', '~> 4.0.1'
-    pod 'IBMWatsonToneAnalyzerV3', '~> 4.0.1'
-    pod 'IBMWatsonVisualRecognitionV3', '~> 4.0.1'
+    pod 'IBMWatsonAssistantV1', '~> 4.0'
```

Comment:
These version numbers are updated by the release automation and shouldn't be hand edited.